app/account/views.py

```diff
@@ -250,3 +251,43 @@ def unconfirmed():
     if current_user.is_anonymous() or current_user.confirmed:
         return redirect(url_for('main.index'))
     return render_template('account/unconfirmed.html')
+
+
+@account.route('/profile/<int:user_id>')
+@login_required
+def profile(user_id):
+    """Display a user's profile"""
+    user = User.query.get(user_id)
+    if user is None:
+        # tried to change to abort(404) but threw errors
```

Comment:
You probably have to say something like `from flask import abort`. Check out [these docs](http://flask.pocoo.org/docs/0.10/quickstart/#redirects-and-errors).